### PR TITLE
Added HITMAPS presence & Updated Netflix icon

### PR DESCRIPTION
**Pull request by QkeleQ10**
Added a presence for HITMAPS™. This is a website that has detailed, interactive maps for the HITMAN™ series.

https://www.hitmaps.com, https://roulette.hitmaps.com

imageimageimageimage

### CODE REVIEW

File: websites/H/HITMAPS/presence.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  clientId: "842704573877714974"
+});
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "app"
+  };
+
+  if (document.location.hostname === "roulette.hitmaps.com") {
+    presenceData.details = "Playing HITMAPS™ Roulette"
```
Comment (ririxidev) [inline websites/H/HITMAPS/presence.ts:11]: about this i think you can do more stuff on this subdomain

File: websites/H/HITMAPS/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
```
Comment (ririxidev) [inline websites/H/HITMAPS/dist/metadata.json:2]:
```suggestion
  "$schema": "https://schemas.premid.app/metadata/1.3",
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/H/HITMAPS/presence.ts (hunk 1 above)
Comment (QkeleQ10) [reply websites/H/HITMAPS/presence.ts:11]: I guess I could add one little thing... Adding that now.

**Review by ririxidev — APPROVED** [review]

**Comment by QkeleQ10** [comment]
Sorry 🥲 Updated the Netflix icon because it was a tad too small and didn't quite fit in

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**